src/utils/motions.ts

```diff
@@ -0,0 +1,59 @@
+import { AnyColonyClient, Extension } from '@colony/colony-js';
+import { TransactionDescription } from 'ethers/lib/utils';
+
+import { mutate } from '../amplifyClient';
+import { formatDomain } from '../domains';
+import { ContractEvent, motionNameMapping } from '../types';
+import { verbose } from './logger';
+import { getColonyTokenAddress, getColonyTokenData } from './tokens';
+
+export const getParsedActionFromMotion = async (
```

Comment:
This is how to get the underlying action associated with the motion, e.g mint tokens